coral/src/app/features/configuration/environments/SchemaRegistry/SchemaRegistryEnvironments.test.tsx

```diff
@@ -59,6 +60,7 @@ const mockedEnvironmentsResponse: EnvironmentPaginatedApiResponse =
       tenantName: "default",
       clusterName: "DEV_CLS",
       envStatus: "ONLINE",
+      envStatusTime: "2023-09-21T11:47:15.664615239",
```

Comment:
I'm wondering if we don't also want a `requesttimestring` property as we have for the`[entity]RequestsResponseModel`. We can parse it on the front end, but then we have a potential for divergence if the formatting changes in one of the two spots we are now doing time formatting in :o

Reply:
I am adding in entStatusRequestedAtTime

Reply:
I'm sorry, I didn't mean to add the time of the request, but havoing the time formatted in the same way as `requesttimestring` 🙇  Maybe the following is already what is happening, but the test values express a different thing: https://github.com/Aiven-Open/klaw/pull/1747/files#diff-df59e5cc730be0ccaca740fec7b48fdf327c4430601001cf99da3813e39f32d7R42

The difference between these two: https://github.com/Aiven-Open/klaw/blob/main/openapi.yaml#L7062-L7068

Is that one is the undformatted timestamp (`requesttime`) and the other is a formatted string expressing the time in the UTC tz (`requesttimestring`)

We are currently doing no parsing of timestamps in the FE, only rendering the already formatted `requesttimestring`:

<img width="1380" alt="Screenshot 2023-09-26 at 13 33 28" src="https://github.com/Aiven-Open/klaw/assets/20607294/a3638fde-0861-418c-bd37-0d52b5e8ec6b">


And I thought we might do the same for the `envStatusTime` timestamp: same time, but formatted differently.